src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -1444,6 +1444,44 @@ public function validateIpv6($attribute, $value)
         return filter_var($value, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6) !== false;
     }
 
+    /**
+     * Validate that an attribute is a valid CIDR.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function validateCidr($attribute, $value)
+    {
+        $parts = explode('/', $value);
+
+        if (count($parts) !== 2) {
+            return false;
+        }
+
+        [$ip, $mask] = $parts;
+
```

Comment:
A suggestion, IMO would be clearer

```php
    if ($this->validateIpv4($attribute, $ip)) {
        return filter_var($mask, FILTER_VALIDATE_INT, ['options' => ['min_range' => 0, 'max_range' => 32]]) !== false;
    }

    if ($this->validateIpv6($attribute, $ip)) {
        return filter_var($mask, FILTER_VALIDATE_INT, ['options' => ['min_range' => 0, 'max_range' => 64]]) !== false;
    }

    return false;
```

Reply:
I like that! Thanks @rodrigopedra ! 
I applied it.